script.js

```diff
@@ -1,18 +1,21 @@
 
 //start the game 
 let PlayerSelection = 'rock';
+let possibleChoices = ['rock', 'paper', 'scissors'];
```

Comment:
later down the road, the function was looking for a `possibleChoices.length`, so I just added in that here